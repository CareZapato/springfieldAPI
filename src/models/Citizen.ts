export interface Citizen {
    name: string;
    lastName: string;
    id: string;
    gender: string;
    birthDate: Date;
    address: string;
    job: string;
    isAlive: boolean;
}