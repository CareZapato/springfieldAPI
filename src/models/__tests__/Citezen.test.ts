import { Citizen } from '../Citizen';

const exampleCitizen: Citizen = {
  name: 'Homer',
  lastName: 'Simpson',
  id: '12345',
  gender: 'Male',
  birthDate: new Date('1956-05-12'),
  address: '742 Evergreen Terrace, Springfield',
  job: 'Safety Inspector',
  isAlive: true
};

describe('Citizen interface', () => {
  it('matches the snapshot', () => {
    expect(exampleCitizen).toMatchSnapshot();
  });
});
