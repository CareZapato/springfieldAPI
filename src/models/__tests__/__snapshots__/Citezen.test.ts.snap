// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Citizen interface matches the snapshot 1`] = `
{
  "address": "742 Evergreen Terrace, Springfield",
  "birthDate": 1956-05-12T00:00:00.000Z,
  "gender": "Male",
  "id": "12345",
  "isAlive": true,
  "job": "Safety Inspector",
  "lastName": "Simpson",
  "name": "Homer",
}
`;
